spec/index.bs

```diff
@@ -0,0 +1,205 @@
+<pre class='metadata'>
+Title: Login Status API
+Shortname: login-status
+Level: 1
+Status: w3c/FPWD
+Group: fedid
+TR: http://www.w3.org/TR/fedcm/login-status.html
+ED: https://w3c-fedid.github.io/fedcm/login-status.html
+Prepare for TR: true
+Repository: w3c-fedid/FedCM
+Editor: Christian Biesinger, w3cid 39461, Google Inc. https://google.com, cbiesinger@chromium.org
+Editor: John Wilander, Apple Inc. https://apple.com, wilander@apple.com
+
+Markup Shorthands: markdown yes, biblio yes
+Default Biblio Display: inline
+
+Text Macro: FALSE <code>false</code>
+Text Macro: TRUE <code>true</code>
+Text Macro: RP Relying Party
+Text Macro: IDP Identity Provider
+
+Abstract: A Web Platform API that allows websites to notify the browser when their users login and logout.
+
+Test Suite: https://github.com/web-platform-tests/wpt/tree/master/fedcm
```

Comment:
add /fedcm-login-status here (sorry, missed this in the initial review)

Reply:
done